upgrade/steps.go

```diff
@@ -837,12 +837,16 @@ func migrationSteps(ctx Context, repo ProviderRepo, providerName string, descrip
 func AddAutoAliasing(ctx Context, repo ProviderRepo, providerName string) (step.Step, error) {
 	steps := []step.Step{}
 	metadataPath := fmt.Sprintf("%s/cmd/pulumi-resource-%s/bridge-metadata.json", *repo.providerDir(), providerName)
-	if _, err := os.Stat(metadataPath); os.IsNotExist(err) {
-		_, err = os.Create(metadataPath)
-		if err != nil {
-			return nil, err
+	steps = append(steps, step.F("initializing bridge-metadata.json", func() (string, error) {
```

Comment:
We could declare the `initializing bridge-metadata.json` step and the `git add metadataPath` step as part of the `steps` literal.

```go
initializeBridgeMetadata := step.F(...)
addBridgeMetadata := step.Cmd(...)
steps := []step.Step{initializeBridgeMetadata, addBridgeMetadata}
```

We could also declare them inline.

```go
steps := []step.Step{ 
	step.F(...),
	step.Cmd(...),
}
```

Lets do one or the other here.